app/services/solidus_bolt/payments/capture_service.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module Payments
+    class CaptureService < BaseService
+      attr_reader :payment, :order, :payment_source
+
+      def initialize(payment:)
+        @payment = payment
+        @order = payment.order
+        @payment_source = payment.payment_source
+        super
+      end
+
+      def call
+        return unless payment_source.can_capture?(payment)
```

Comment:
I don't think we should check it. 
Considering that Bolt is the source of truth if it says that the payment has been captured, we just have to move the payment to the capture state.

Reply:
Following the above conversion, maybe we just have to check if the payment is already captured to avoid capturing an already captured payment.
It will happen whenever the payment will be captured from the solidus side, and the webhook is enabled.